src/Helpmebot/Helpmebot.cs

```diff
@@ -154,6 +154,7 @@ private static void InitialiseBot()
 
             if (!dbal.Connect())
             {
+				Log.Error("Unable to connect to the DB on INIT");
```

Comment:
The actual error is logged out already in the connect message with the error severity, this will duplicate the production alerting that's configured (which is emailing me every time an error is logged), with the second alert containing almost no information.

Oh, four spaces instead of tabs too please :)